roles/configure_ztp_gitops_apps/tasks/main.yaml

```diff
@@ -208,6 +208,22 @@
             .dockerconfigjson: "{{ _czga_encoded_pull_secret['content'] }}"
       no_log: true
 
+    - name: Wait for the GitOps ZTP plugin activation
+      community.kubernetes.k8s_info:
+        api_version: v1
+        kind: Pod
+        namespace: openshift-gitops
+        label_selectors:
+          - app.kubernetes.io/name = openshift-gitops-repo-server
+      register: repo_server_pod
```

Comment:
this var needs to be renamed to fit in ansible lint check: `-ERROR: roles/configure_ztp_gitops_apps/tasks/main.yaml:: redhat-ci[no-role-prefix]: Variable names should use a prefix related to the role. ({'_configure_ztp_gitops_apps', '_czga'} are possible) (register: repo_server_pod)
`